index.html

```diff
@@ -187,26 +187,27 @@ <h2 class="about-grid-item-title">Skills</h2>
         </div>
       </div>
     </div>
-    <div id="contact-div">
-      <div id="contact-center">
-        <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
-        <div id="contact-form-div">
-          <form id = "survey-form">
-            <input id="fullname" class="form name" type="text" placeholder="Full Name" required>
-            <input id="first-name" class="form name desktop" type="text" placeholder="First Name" required><input id="last-name" class="form name desktop" type="text" placeholder="Last Name" required><br>
-            <input id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
-            <textarea id="comments" class="form" name="comments" placeholder="Hey I’d like to connect to..." required></textarea><br>
-            <div id="submit-button" class="grid-button-container">
-              <input type="submit" id="submit" class="grid-button" value="Get in touch">
-            </div>
-            <div id="email-para">
-              <p>nyldrift@hotmail.com</p>
-            </div>
-          </form>
+  </section>
+  <section id="contact">
+    <div id="contact-center">
+      <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+      <div id="contact-form-div">
+        <form id = "survey-form" action="https://formspree.io/f/mzbqjbpr" method="POST">
+          <input name="First Name" id="first-name" class="form name" type="text" placeholder="First Name" maxlength="30" required>
+          <input id="Last Name" class="form name desktop" type="text" placeholder="Last Name" required><br>
+          <input name="Email" id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
+          <textarea name="Comments" id="comments" class="form" placeholder="Hey I’d like to connect to..." maxlength="500" required></textarea><br>
+          <div id="submit-button" class="grid-button-container">
+            <input type="submit" id="submit" class="grid-button" value="Get in touch">
+          </div>
+        </form>
+        <div id="email-para">
+          <p>nyldrift@hotmail.com</p>
         </div>
       </div>
     </div>
-    </section>
+  </div>
+  </section>
```

Comment:
- [ ] According to the project requirements we must stick to the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=1yfXx6yoiCqxC0tL-0) and make sure our design looks identical to figma. So please make sure your design also includes all the elements present in the figma design as your design is missing a line under the footer icons and the background image.

Figma design

<img width="432" alt="Screenshot 2023-03-29 at 8 49 10 AM" src="https://user-images.githubusercontent.com/44798044/228424135-e74a55aa-ebdf-43ef-a7ba-21799b263f9c.png">

Your design:

<img width="307" alt="Screenshot 2023-03-29 at 8 48 49 AM" src="https://user-images.githubusercontent.com/44798044/228424234-e136a552-b41d-4f39-a9cb-f6c6a6d3cfb8.png">

Reply:
- background image added again
- line footer wont be added as it makes no sense as the line is supose to be part of the phone system, like the battery and hour, wich i wont include for obvious reasons.
I included example of a live view of the website, as it seems you aren't familiarized with mobile phone UI:
![image](https://user-images.githubusercontent.com/16127046/228652502-e3e4b228-9481-4a2f-a204-874f770bc59f.png)